### ISSUE TITLE
Publish a new version of NPM package containing recent fixes?

### ISSUE THREAD

**Issue by w1ndy** [sigma-geosistemas/Leaflet.awesome-markers#11]
It has been a while since last release. Important fixes like #6 were not included, making webpack unable to bundle the package correctly.

**Comment by w1ndy** [comment]
Temporary workaround for webpack:

```javascript
{
  resolve: {
    alias: {
      'leaflet.awesome-markers$': 'leaflet.awesome-markers/dist/leaflet.awesome-markers.js'
    }
  }
}
```

**Comment by AlexGreg** [comment]
fixed with [2.0.5](https://github.com/sigma-geosistemas/Leaflet.awesome-markers/releases/tag/2.0.5) release
